Lab3/Weather/Map.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
```

Comment:
there is no mobile version here (
<img width="689" alt="Screenshot 2022-04-05 at 14 12 19" src="https://user-images.githubusercontent.com/43992068/161751039-6e300bdf-8e81-4453-b627-be39b5ccbdc2.png">
also, smth is wrong with nav 
<img width="686" alt="Screenshot 2022-04-05 at 14 12 43" src="https://user-images.githubusercontent.com/43992068/161751107-6158e332-e22e-4b03-9f5a-c453278b7204.png">